README.md

```diff
@@ -155,23 +141,26 @@ Docker image, pushing code staging:
 
 This application is deployed to [Cloud Platform](https://user-guide.cloud-platform.service.justice.gov.uk/).
 
-Currently we have a `staging` and `production` environment on Cloud
+Currently we have a `dev`, `staging`, `uat`, `preprod` and `production` environment on Cloud
 Platform with the following namespace names:
 
+* hmpps-book-secure-move-api-dev
 * hmpps-book-secure-move-api-staging
+* hmpps-book-secure-move-api-uat
+* hmpps-book-secure-move-api-preprod
 * hmpps-book-secure-move-api-production
 
-`staging` that is automatically deployed on successful builds of the
+`dev` and `staging` environments are automatically deployed on successful builds of the
 `master` branch on Circle CI.
 
-`production` can be deployed on any successful build of `master` but
-requires manual intervention to approve the `hold_production` step in
-the workflow. To approve a particular deploy for build log into
-CircleCI, find the build that you want to deploy and click the
-`test-build-deploy` link. This should take you to the workflow graph for
-that build where you can click the `hold_production` step to kick off
-the production deploy.
+`preprod`, `uat` and `production` can be deployed by generating a new tag
+following [semantic versioning](https://semver.org/) pointing to the current commit.
```

Comment:
```suggestion
following [semantic versioning](https://semver.org/) pointing to the current commit. We typically do this locally with:

```bash
# Get a list of tags
git tag
# Tag following semantic versioning rules
git tag v1.0.1
# Push the tag to origin
git push origin v1.0.1
```

```